src/styles/index.scss

```
@@ -1,5 +1,5 @@
 body {
-  font-family: 'montserrat', sans-serif;
+  font-family: "montserrat", sans-serif;
```

Stylelint (17.11.0)

[error] Expected no quotes around "montserrat" (font-family-name-quotes)

(font-family-name-quotes)